ADOPTERS.md

```diff
@@ -1,15 +1,65 @@
-* [Radio Sound](https://radiosound.com/) uses Rook to power their website and GitLab for their CI/CD pipeline, because of the truly cloud-native experience, like *"a little drop of the Google magic in our own server rack"*.
+* [Calit2 (California Institute for Telecommunications and Information
+  Technology)](http://www.calit2.net/) is one of 4 institutes formed by a joint partnership of
+  University of California and the state of California with the goal of *“inventing the university
+  research environment of the future”*.  They operate one of the largest known Rook clusters in
+  production and they are using Rook to provide cheap, reliable, and fast storage to scientific
+  users.
+* [NAV (Norwegian Labour and Welfare Administration)](https://www.nav.no/) is the current Norwegian
+  public welfare agency, responsible for 1/3 of the state budget of Norway. They find a massive
+  simplification of management and maintenance for their Ceph clusters by adopting Rook.
+* [Replicated](https://www.replicated.com/) delivers *“SaaS On-Prem”* and are the creators of
+  open-source [kURL](https://kurl.sh/): a custom Kubernetes distro creator that software vendors use
+  to package and distribute production-grade Kubernetes infrastructure. Rook is a default add-on in
+  kURL, so all installations include Rook to manage highly available storage that the software
+  vendor can depend on.
+* [Discogs](https://www.discogs.com/) is building the largest and most comprehensive music database
+  and marketplace in the world and services millions of users all across the globe. Rook enables
+  them to save both time and money in the long term and allows their IT operations team to function
+  with fewer dedicated staff.
+* [Finleap Connect](https://connect.finleap.com/) offers a full range of leading fintech solutions
+  to financial institutions across Europe. Rook has been running flawlessly for them across many
+  versions and upgrades, and delivers the performance and resilience they require for their most
+  critical business applications.
+* [Centre of Excellence in Next Generation Networks (CENGN)](https://www.cengn.ca) is on a mission
+  to accelerate the growth of the Canadian Information and Communications Technology (ICT) sector.
+  The Rook Ceph operator is key to the Kubernetes clusters frequently set up for projects by
+  small-medium enterprises in the CENGN labs.
+* [Avisi](https://www.avisi.nl/) develops and operates software for organizations like the the Dutch
+  Notary Association. They have survived multiple disaster scenarios already with Rook and it has
+  made their cloud native journey in the private cloud so much easier by providing a powerful tool
+  that lets them take advantage of a mature storage product with ease and peace of mind.
+* [Geodata](https://geodata.no/) Provides geospatial services and Geographical Information Systems
+  (GIS).  The latest versions of Rook has amazed them, especially compared to many of the other
```

Comment:
"The latest versions of Rook have amazed..."

Reply:
Fixed this typo